data-serving/data-service/src/controllers/case.ts

```diff
@@ -284,13 +283,10 @@ export const batchUpsert = async (
             { ordered: false },
         );
         res.status(207).json({
-            // Types are a little goofy here. We're grabbing the string ID from
-            // what MongoDB returns, which is data in the form of:
-            //   { index0 (string): _id0 (ObjectId), ..., indexN: _idN}
-            createdCaseIds: Object.entries(bulkWriteResult.insertedIds)
-                .concat(Object.entries(bulkWriteResult.upsertedIds))
-                .map((kv) => String(kv[1])),
-            updatedCaseIds: toBeUpsertedCaseIds,
+            numCreated:
+                (bulkWriteResult.insertedCount || 0) +
```

Comment:
I thought the fields on BulkWriteResults were `nInserted` and `nUpserted`? https://docs.mongodb.com/manual/reference/method/BulkWriteResult/

Reply:
It's a `BulkWriteOpResult`, which is slightly different:

http://mongodb.github.io/node-mongodb-native/3.1/api/Collection.html#~BulkWriteOpResult
https://mongoosejs.com/docs/api/model.html#model_Model.bulkWrite